tutorials/layout/main.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+
+void main() {
+  //debugPaintSizeEnabled = true;
+  runApp(new MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    Widget titleSection = new Container(
+      padding: const EdgeInsets.all(32.0),
+      child: new Row(
+        children: [
+          new Expanded(
+            child: new Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                new Container(
+                  padding: const EdgeInsets.only(bottom: 8.0),
+                  child: new Text(
+                    'Oeschinen Lake Campground',
+                    style: new TextStyle(
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
+                ),
+                new Text(
+                  'Kandersteg, Switzerland',
+                  style: new TextStyle(
+                    color: Colors.grey[500],
+                  ),
+                ),
+              ],
+            ),
+          ),
+          new Icon(
+            Icons.star,
+            color: Colors.red[500],
+          ),
+          new Text('41'),
+        ],
+      ),
+    );
+
+    Column _buildButtonColumn(IconData icon, String label) {
+      Column buttonColumn;
+      List<Widget> widgets = new List(2);
+
+      widgets[0] = new Icon(icon, color: Colors.yellow[600]);
+      widgets[1] = new Container(
+        margin: const EdgeInsets.only(top: 8.0),
+        child: new Text(
+          label,
+          style: new TextStyle(
+            fontSize: 12.0,
+            fontWeight: FontWeight.w400,
+            color: Colors.yellow[600],
+          ),
+        ),
+      );
+
+      buttonColumn = new Column(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
```

Comment:
This is the default value.  Not sure if you're specifying it explicitly on purpose.